prometheus-exporters/ucs-exporter/alerts/ucsmcrcerrors.alerts

```diff
@@ -2,13 +2,15 @@ groups:
 - name: ucsmcrcerrors.alerts
   rules:
   - alert: UcsmCRCError
-    expr: max_over_time(ucsm_crc_error[20m]) > 0
+    expr: increase(ucsm_crc_error[30m]) > 100
+    for: 20m
     labels:
       severity: info
       tier: vmware
       service: compute
       context: "ucs-exporter"
       meta: "Server {{ $labels.server }} has crc errors for port {{ $labels.port }}"
+      no_alert_on_absence: "true"
```

Comment:
While this disabled the absent alerts, it does not solve the root cause. I would suggest lowering the scrape interval and make sure that there is always a metric in the alert expression range available.

Reply:
Please remove, the absent alerts are actually pretty valuable.